crates/builtin-binaries/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "gw-builtin-binaries"
+version = "1.8.0-rc3"
+edition = "2021"
+authors = ["Godwoken"]
+license = "MIT"
+description = "Godwoken builtin binaries"
+repository = "https://github.com/godwokenrises/godwoken"
+include = ["src/**/*", "Cargo.toml", "build.rs", "builtin/**/*", "README.md"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+default = []
+no-builtin = []
```

Comment:
Cargo features are supposed to be additive, so it might be better to have a `builtin` feature to include builtin scripts rather than having a `no-builtin` feature.

Also, would it be better to default to not having built-in scripts, and only embed built-in scripts on release?

Reply:
Since builtin binaries would be the recommended way to start a Godwoken node, I think we should make it default.

I will update the code to use active style feature name.

Reply:
Does this mean that we have to build a special version/image for Godwoken internal alphanet?

Reply:
It's an unrelated topic. But yes.